tests/conftest.py

```diff
@@ -75,6 +76,11 @@ def small_sky_order1_dir(test_data_dir):
     return test_data_dir / SMALL_SKY_ORDER1_DIR_NAME
 
 
+@pytest.fixture
+def small_sky_order1_no_pandas_dir(test_data_dir):
+    return test_data_dir / SMALL_SKY_ORDER1_NO_PANDAS_DIR_NAME
```

Comment:
Note that with https://github.com/astronomy-commons/hats-import/pull/415, this behavior will be the default.

Reply:
Got it! I think when that's published we can regenerate the lsdb test data and I'll change this to test with the pandas metadata